CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Previous versions had CHANGELOG per component, we decided to combine all reposit
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
 ## [Unreleased]
+### Added
+- Add support for openat, closes [#120](https://github.com/metalbear-co/mirrord/issues/120)
```

Comment:
Is it just openat? Because if so it doesn't close #120, which specifies ls as well. Maybe edit #120 to only specify openat, and open a separate issue for ls?

Reply:
It will be for both, just a draft for now. Havent pushed all changes